docs/notebooks/Common_Gotchas_in_JAX.md

```diff
@@ -532,333 +532,6 @@ for subkey in subkeys:
   print(random.normal(subkey, shape=(1,)))
 ```
 
-+++ {"id": "rg4CpMZ8c3ri"}
-
-## 🔪 Control flow
```

Comment:
Let's keep this heading here and under it say `moved to ...`.

We should also redirect the handful of places this heading is referred to by URL: https://github.com/search?q=repo%3Ajax-ml%2Fjax+%22JAX.html%23control-flow%22&type=code

Reply:
Also some of the other headings below, e.g. https://github.com/search?q=repo%3Ajax-ml%2Fjax+%22JAX.html%23python%22&type=code